src/test/java/teammates/test/cases/action/StudentProfilePictureActionTest.java

```diff
@@ -335,9 +334,10 @@ protected StudentProfilePictureAction getAction(String... params) {
         return (StudentProfilePictureAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
+    @Test
     @Override
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        testActionWithEmailAndCourse();
```

Comment:
In `testActionWithEmailAndCourse`, only `testActionWithEmailAndCourseUnauthorisedInstructorOrStudent` and `testActionWithEmailAndCourseUnauthorisedInstructorOrStudentMasquerade` are more relevant to testing of access control. So `testActionWithEmailAndCourse` should not be moved entirely to `testAccessControl`.